sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -572,7 +572,7 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
             if (useYarn.value) {
               Yarn.run("install", "--non-interactive")(targetDir, log)
             } else {
-              Npm.run("update")(targetDir, log)
+              Npm.run("install", ".")(targetDir, log)
```

Comment:
What’s the benefit of the `.`?

Reply:
Just explicitness:

Reply:
That said, I glazed over the first line when I was reading that help the first time, so it's apparently documented behavior to look for package.json in the current folder.